mne/chpi.py

```diff
@@ -443,19 +443,13 @@ def _setup_hpi_amplitude_fitting(info, t_window, remove_aliased=False,
     # grab basic info.
     hpi_freqs, hpi_pick, hpi_ons = _get_hpi_info(info, allow_empty=allow_empty)
     _validate_type(t_window, (str, 'numeric'), 't_window')
-    if isinstance(t_window, str):
-        if t_window != 'auto':
-            raise ValueError('t_window must be "auto" if a string, got %r'
-                             % (t_window,))
-        if len(hpi_freqs):
-            t_window = max(5. / min(hpi_freqs), 1. / np.diff(hpi_freqs).min())
-        else:
-            t_window = 0.2
-    t_window = float(t_window)
-    if t_window <= 0:
-        raise ValueError('t_window (%s) must be > 0' % (t_window,))
-    logger.info('Using time window: %0.1f ms' % (1000 * t_window,))
-    model_n_window = int(round(float(t_window) * info['sfreq']))
+    if info['line_freq'] is not None:
+        line_freqs = np.arange(info['line_freq'], info['sfreq'] / 3.,
+                               info['line_freq'])
+    else:
+        line_freqs = np.zeros([0])
```

Comment:
Won't this add an additional DC term in the model? Seems like it would be better to use just:
```suggestion
        line_freqs = np.empty(0)
```
i.e., an empty zero-element array. The concatenations and such should still work. The `logger.info` line might need to be adjusted, though

Reply:
I think that line was intended to do exactly what you suggest: give an empty array (np.zeros[0]) doesn't fill with a zero, but the resulting array has zero size).  Your way is more explicit.

Reply:
Argh you're right -- either way is fine by me